src/pyseekdb/utils/embedding_functions/text2vec_embedding_function.py

```
@@ -0,0 +1,128 @@
+                from text2vec import SentenceModel
+                # Initialize the model
+                self.models[model_name] = SentenceModel(
+                    model_name_or_path=model_name,
+                    device=device,
+                    **kwargs
+                )
+            except ImportError as exc:
+                raise ValueError(
+                    "The text2vec python package is not installed. "
+                    "Please install it with: `pip install text2vec`"
+                ) from exc
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)